docs/reference/elaborator-reflection.rst

```diff
@@ -74,17 +74,54 @@ We can generate identity functions at any concrete type using the same script:
 Interactively Building Elab Scripts
 ===================================
 
-To build an ``Elab`` script interactively, use the ``:elab`` command at the REPL.
-It takes the name of a hole as an argument.
-To list all available holes, use the command ``:m``.
-
-In interactive elaboration shell, the following commands are available:
--  ``:q`` - Quits the elaboration shell (gives up on proving current lemma).
--  ``:abandon`` - Same as :q
--  ``:state`` - Displays the current state of the term being constructed.
--  ``:term`` - Displays the current proof term complete with its yet-to-be-filled holes.
--  ``:undo`` - Undoes the last tactic.
--  ``:qed`` - Once the elaboration shell tells you "No more goals," you get to type this in celebration! (Completes the construction and exits the shell)
+You can build an ``Elab`` script interactively at the REPL.
+Use the command ``:metavars``, or ``:m`` for short, to list the available holes.
+Then, issue the ``:elab <hole>`` command at the REPL
+to enter the elaboration shell.
+
+At the shell, you can enter proof tactics to alter the proof state.
+It is recommended to issue ``:module Language.Reflection.Elab``
```

Comment:
Gotcha. For some reason, I thought docs updates were more frequent, maybe even tracking master.
--
Jeremy W. Sherman
http://jeremywsherman.com/

> El 12-10-2015, a las 12:53, David Christiansen <notifications@github.com> escribió:
> 
> In docs/reference/elaborator-reflection.rst:
> 
> > -To list all available holes, use the command ``:m``.
> > -
> > -In interactive elaboration shell, the following commands are available:
> > --  ``:q`` - Quits the elaboration shell (gives up on proving current lemma).
> > --  ``:abandon`` - Same as :q
> > --  ``:state`` - Displays the current state of the term being constructed.
> > --  ``:term`` - Displays the current proof term complete with its yet-to-be-filled holes.
> > --  ``:undo`` - Undoes the last tactic.
> > --  ``:qed`` - Once the elaboration shell tells you "No more goals," you get to type this in celebration! (Completes the construction and exits the shell)
> > +You can build an ``Elab`` script interactively at the REPL.
> > +Use the command ``:metavars``, or ``:m`` for short, to list the available holes.
> > +Then, issue the ``:elab <hole>`` command at the REPL
> > +to enter the elaboration shell.
> > +
> > +At the shell, you can enter proof tactics to alter the proof state.
> > +It is recommended to issue ``:module Language.Reflection.Elab``
> This will be true in the next release, which is also the release that the documentation in the repo should pertain to :)
> 
> —
> Reply to this email directly or view it on GitHub.
>